app/models/user.rb

```diff
@@ -9,4 +9,44 @@ class User < ApplicationRecord
   has_many :posts
```

Comment:
Remember that the Timeline has to show the user's posts **AND** also the posts of their friends.

Reply:
Implemented